Crypto/PublicKey+Apple.mm

```diff
@@ -595,8 +595,10 @@ virtual int _sign(int/*mbedtls_md_type_t*/ mbedDigestAlgorithm,
             OSStatus err;
             if (@available(iOS 12.0, macos 10.14, *)) {
                 CFErrorRef cferr;
-                if (!SecTrustEvaluateWithError(trustRef, &cferr))
-                    warnCFError(cferr, "SecTrustEvaluateWithError");
+                if (!SecTrustEvaluateWithError(trustRef, &cferr)) {
+                    auto error = (__bridge NSError*)cferr;
+                    LogTo(TLSLogDomain, "SecTrustEvaluateWithError %s", error.description.UTF8String);
```

Comment:
This will not log to CBL log?

Reply:
No, it will not log. I guess the TLSLogDomain is not getting logged as part of Sync. I will check it into more details.

Reply:
Spend couple of hours to investigate but didn't find the root cause why not logging, will further check in separate task CBL-2778